packages/react-tabster/src/hooks/useKeyborg.ts

```diff
@@ -0,0 +1,38 @@
+import { createKeyborg } from 'keyborg';
+import { KeyborgCallback } from 'keyborg/dist/Keyborg';
+import { RefObject, useEffect, useMemo, useRef } from 'react';
+import { KEYBOARD_NAV_ATTRIBUTE } from '../symbols';
+import { useConst } from '@fluentui/react-utilities';
+import { useFluent } from '@fluentui/react-shared-contexts';
+
+/**
+ * instantiates keyborg and add attribute to ensure focus indicator synced to keyborg logic
+ */
+export function useKeyborg<E extends HTMLElement>() {
```

Comment:
returning `refs` seems a bit complicated to me of an API, because then it requires merging if the consumer also wants their own ref.

I kind rather thought this would just return s `subscribe` callback that would accept a HTML element and then let the consumer worry about refs

Reply:
> I kind rather thought this would just return s subscribe callback that would accept a HTML element and then let the consumer worry about refs

That's pretty  much `ref` as a function definition 😅.

Well, IMO since we also provide an easy way of merging refs, I don't see this as a big problem, afterall the user **should** merge refs in scenarios like this.

Also this is a common approach. Other APIs lke [react-hook-form](https://react-hook-form.com/) rerturns a reference for every registered input in their main hook.

Reply:
Yeah the merging refs is kinda new to me, but if it's a common scenario that's fine with me 👍